### DISCUSSION TITLE
can this software be used on the heart?

### DISCUSSION THREAD

**Discussion by strawberry1996** [SCIInstitute/SCIRun#2035] [Q&A] [answered]
I want to use the ex vivo cardiac DWI to simulate the in vivo cardiac DWI by this software, but I see the software only has simulation tools for the brain

**Comment by dcwhite** [comment]
Heart is definitely supported, there are many examples and tutorials. @jessdtate can answer your more specific questions.

**Comment by jessdtate** [comment, marked as answer]
there is a tutorial that shows how to use conductivity tensors in hearts to simulate ischemia (https://github.com/SCIInstitute/scirun.pages/blob/ischemia-tutorial/IschemiaTutorial.md, or for the old version http://www.sci.utah.edu/devbuilds/scirun_docs/IschemiaModelTutorial.pdf).  There is also a fwd/inv toolkit that has many examples that show how to use cardiac data in torso models (https://github.com/SCIInstitute/FwdInvToolkit).  You can try the scirun-users@sci.utah.edu mailing list if you have more specific questions.

**Comment by jessdtate** [comment]
@strawberry1996 Did this information help you?
